tests/integration/test_composable_protocols/configs/users.xml

```diff
@@ -0,0 +1,18 @@
+<clickhouse>
+    <profiles>
+        <default>
+            <max_memory_usage>10000000000</max_memory_usage>
```

Comment:
Why do we need these settings?

Reply:
it's just a copy-paste from some other test - does nothing

Reply:
If it's not necessary, let's remove it?